packages/sequence-diagram/appmap.yml

```diff
@@ -0,0 +1,45 @@
+ordering: chronological
+command: yarn exec jest --no-cache
+recorder: jest
+exclude:
+- \[anonymous\]
+- matchAction
```

Comment:
Any reason why this is not scoped to a single package?

Reply:
No reason, thanks.